Source/SourceKittenFramework/SwiftDeclarationAttributeKind.swift

```diff
@@ -95,4 +95,21 @@ public enum SwiftDeclarationAttributeKind: String {
     // only available in Swift < 4.1
     case autoclosure = "source.decl.attribute.autoclosure"
     case noescape = "source.decl.attribute.noescape"
+
+    // only available in Swift >= 5.0
+    case __raw_doc_comment = "source.decl.attribute.__raw_doc_comment"
+    case __setter_access = "source.decl.attribute.__setter_access"
+    case _borrowed = "source.decl.attribute._borrowed"
+    case _dynamicReplacement = "source.decl.attribute._dynamicReplacement"
+    case _effects = "source.decl.attribute._effects"
+    case _hasInitialValue = "source.decl.attribute._hasInitialValue"
+    case _hasStorage = "source.decl.attribute._hasStorage"
+    case _nonoverride = "source.decl.attribute._nonoverride"
+    case _private = "source.decl.attribute._private"
+    case _show_in_interface = "source.decl.attribute._show_in_interface"
+    case dynamicCallable = "source.decl.attribute.dynamicCallable"
+    // lower case variants are already exists
+    case gkInspectable = "source.decl.attribute.GKInspectable"
```

Comment:
The lowercase versions already exists:
```swift
    case gkinspectable = "source.decl.attribute.gkinspectable"
    case ibaction = "source.decl.attribute.ibaction"
    case iboutlet = "source.decl.attribute.iboutlet"
```
But I added camel case versions separately.
I have not confirmed whether SourceKit returns the camel case version.

Reply:
I'll drop them from `SwiftDeclarationAttributeKind`

Reply:
Removed on e43887a